samples/sampler/stories/default/ImageItem.js

```diff
@@ -1,13 +1,14 @@
 import {mergeComponentMetadata} from '@enact/storybook-utils';
 import {action} from '@enact/storybook-utils/addons/actions';
-import {boolean, object, text, select} from '@enact/storybook-utils/addons/controls';
+import {boolean, text, select} from '@enact/storybook-utils/addons/controls';
 import ri from '@enact/ui/resolution';
 import ImageItem, {ImageItemBase} from '@enact/agate/ImageItem';
 
 const src = {
-	'hd':  'http://via.placeholder.com/200x200/9037ab/ffffff&text=Image0',
-	'fhd': 'http://via.placeholder.com/300x300/9037ab/ffffff&text=Image0',
-	'uhd': 'http://via.placeholder.com/600x600/9037ab/ffffff&text=Image0'
+	hd:  'http://via.placeholder.com/200x200/9037ab/ffffff&text=Image0',
+	fhd: 'http://via.placeholder.com/300x300/9037ab/ffffff&text=Image0',
+	uhd: 'http://via.placeholder.com/600x600/9037ab/ffffff&text=Image0',
+	invalidUrl: 'ttp://via.placeholder.com/300x300/9037ab/ffffff&text=Image0'
```

Comment:
Hm... How about use other http adress 
https://via.placeholder.com/test ?

Reply:
It is matched with test step's url.